libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts

```diff
@@ -316,3 +327,14 @@ export class DrillConfigItemDrillToUrlDropdown {
         this.getElement().find(".s-drill-to-custom-url-option").contains(url.slice(0, 50)).should("exist");
     }
 }
+
+export class CustomURLDialog {
+    getElement() {
+        return cy.get(".s-gd-drill-custom-url-editor");
+    }
+
+    hasAttribute(title: string) {
```

Comment:
only Attribute can use this function??
If metric/... can use this function also, rename it